Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM python:latest
```

Comment:
Specify exact version instead of latest.

Reply:
i'll fix it in next commit

Reply:
fixed in 727e6d8bef5c9ee39bc4b996a51718e87a6d434a